src/main/java/org/folio/services/invoice/InvoicePaymentService.java

```diff
@@ -43,6 +46,8 @@ public class InvoicePaymentService {
   private CurrentFiscalYearService currentFiscalYearService;
 
   public static final String INVOICE_LINE_MUST_HAVE_FUND = "The invoice line must contain the fund for payment";
+  public static final Set<CompositePoLine.PaymentStatus> PO_LINE_PAYMENT_IGNORED_STATUSES =
```

Comment:
Please use `EnumSet` instead of `Set`. For few element collections AFAIR even `List` is faster than `Set`.

Reply:
Agreed, will switch to `EnumSet`